base/db/tests/testthat/test-get.trait.data.pft.R

```diff
@@ -1,16 +1,8 @@
 context("get.trait.data.pft")
 
-con <- tryCatch(db.open(params = list(
-  user = "bety",
-  password = "bety",
-  host = "localhost"
-)), error = function(e) {
-  PEcAn.logger::logger.severe(paste0(
-    "Failed to open local database connection with the following error: `",
-    conditionMessage(e),
-    "`. Database connection required to run these tests."
-  ))
-})
+source("db.setup.R")
```

Comment:
I trust you're doing this for a good reason, but I don't follow it -- Why does this need to be sourced rather than named as helper-*.R for automatic loading?

Reply:
Didn't know about `helper-*.R` files, but I agree that that's a better approach. I'll change it accordingly.

Reply:
Fixed in e91548baeed4dbf40daf737599f6e24c49d662eb.